docs/git-node.md

```diff
@@ -336,7 +338,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
+Update or patch the V8 engine.
```

Comment:
I'm not sure we want to include those changes (the two spaces are there to force a line break), in any case it's unrelated to the rest of the PR